### PR TITLE
feat: add htop completion spec

### PR DESCRIPTION
**Pull request by jhrcook**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
New feature: add htop completion spec

**What is the current behavior? (You can also link to an open issue here)**
There is no current behavior.

**What is the new behavior (if this is a feature change)?**
The new behavior is to show completion options for the `htop` command.

**Additional info:**
None.

### CODE REVIEW

File: src/htop.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        isOptional: false,
+      },
+    },
+    {
+      name: ["--highlight-changes", "-H"],
+      description: "Highlight new and old processes",
+      args: {
+        name: "delay",
+        isOptional: true,
+      },
```
Comment (fedeci) [inline src/htop.ts:37]: is this supposed to be here?

**Review by fedeci — APPROVED** [review]

File: src/htop.ts (hunk 1 above)
Comment (jhrcook) [reply src/htop.ts:37]: Yes,  it is an optional delay for argument for how frequently to update the highlighting. Should I add a description to that argument to explain that?

Comment (fedeci) [reply src/htop.ts:37]: Yes please!

Comment (jhrcook) [reply src/htop.ts]: Done

File: src/htop.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
+const completionSpec: Fig.Spec = {
+  name: "htop",
+  description: "Improved top (interactive process viewer)",
+
```
Comment (QuiiBz) [inline src/htop.ts:4]: Extra line not needed:
```suggestion
```

File: src/htop.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    {
+      name: ["--no-color", "-C"],
+      description: "Use a monochrome color scheme",
+    },
+    {
+      name: ["--delay", "-d"],
+      description: "Delay between updates, in tenths of sec",
+      args: {
+        name: "delay",
+        suggestions: ["10"],
```
Comment (QuiiBz) [inline src/htop.ts:19]: Maybe we could add 1 and 60?

File: src/htop.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        name: "filter",
+        isOptional: false,
+      },
+    },
+    {
+      name: ["--highlight-changes", "-H"],
+      description: "Highlight new and old processes",
+      args: {
+        name: "delay",
+        description: "Delay between updates of highlights, in tenths of sec",
```
Comment (QuiiBz) [inline src/htop.ts:36]: Could we add the same suggestions as above?

File: src/htop.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    {
+      name: ["--no-mouse", "-M"],
+      description: "Disable the mouse",
+    },
+    {
+      name: ["--pid", "-p"],
+      description: "Show only the given PIDs",
+      args: {
+        name: "PID",
+        isOptional: false,
```
Comment (QuiiBz) [inline src/htop.ts:49]: You can remove `isOptional` when it's `false`. Same for other places underneath

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for this PR! Left few comments before merging it :)

**Review by QuiiBz — APPROVED** [review]
LGTM, thank you so much for this contribution 🚀

**Pull request merged**